src/components/TabBar.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="tab-bar">
+    <a
```

Comment:
```suggestion
    <ButtonPlain
```
would be accurate to use it because, actually, this is not a link